pkg/query-service/app/integrations/messagingQueues/kafka/translator.go

```diff
@@ -12,6 +11,9 @@ var defaultStepInterval int64 = 60
 
 func BuildQueryRangeParams(messagingQueue *MessagingQueue, queryContext string) (*v3.QueryRangeParamsV3, error) {
 
+	if constants.KafkaSpanEval == "false" {
```

Comment:
Is this the right place? It seems like everything else (other scenarios) also fails if it it set to false.